aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/widget/RadioGroupHandler.java

```diff
@@ -38,11 +38,11 @@
     @Override
     public void accept(Element element, Field field) {
         RadioGroup radioGroup = field.getDeclaredAnnotation(RadioGroup.class);
-        if(ArrayUtils.isNotEmpty(radioGroup.buttons())){
+        if (ArrayUtils.isNotEmpty(radioGroup.buttons())) {
```

Comment:
Very nice you don't pass by these small detail. Accuracy is really appreciated. No irony!))